packages/playground/src/constants/settings.js

```diff
@@ -92,6 +92,8 @@ export const stylesBySection = {
         'titlePlacement',
         'overlayBackground',
         'allowSlideshowCounter',
+        'slideshowInfoPlacement',
+        'slideshowInfoWidth',
```

Comment:
dont we already have slidehow info height? why not make it the same styleParam?
It will only ever have one of them. so it can be slidehowInfoSize